### PR TITLE
Add docker support for building

### PR DESCRIPTION
**Pull request by khancyr**
Add support for docker
add instruction for docker
Correct Sphinxsetup for debian
add pycharm as ide for visualisation

### CODE REVIEW

File: common/source/docs/common-wiki-editing-setup.rst [hunk 1]
```diff
@@ -98,6 +99,28 @@ With your favourite web browser, open the locally built wiki which should be nea
 - For Rover look for ``ardupilot_wiki/rover/build/html/index.html``
 - For Developer look for ``ardupilot_wiki/dev/build/html/index.html``
 
+Setup with docker
+-----------------
+
+#. Download and install Docker according to their `official documentation <https://docs.docker.com/install/>`__ .
+
+#. Open a command prompt in the root of the ardupilot_wiki repo and build the docker container for the wiki:
```
Comment (OlivierJB) [inline common/source/docs/common-wiki-editing-setup.rst]: Needs to be a Powershell command prompt on windows. Old DOS command prompt crashes on path in docker run -v option:

docker run --rm -it -v "${PWD}:/ardupilot_wiki" -u "$(id -u):$(id -g)" ardupilot_wiki python update.py
docker: Error response from daemon: create ${PWD}: "${PWD}" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.

Comment (OlivierJB) [reply common/source/docs/common-wiki-editing-setup.rst]: Nice, works great!

**Comment by khancyr** [comment]
@OlivierJB Could you work up the powershell command ? I don't have windows computer to try it ...

**Comment by OlivierJB** [comment]
It just works with Powershell given how it parses file paths (as opposed to dos command prompt), so no need to change anything.

**Comment by khancyr** [comment]
Then I will update to add a comment about windows and powershell !

**Pull request merged**